### PR TITLE
Operator/tpath refactor & Unit test additions

### PR DESCRIPTION
**Pull request by adiprerepa**
[X] Configuration Infrastructure

[X] Does not have any changes that may affect Istio users.

Refactors the unit test structure in `operator/tpath` and adds unit tests for `utils.go`.

### CODE REVIEW

File: operator/pkg/tpath/struct_test.go [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+// limitations under the License.
+
+package tpath
+
+import (
+	"github.com/ghodss/yaml"
+
+	"testing"
+
+	"istio.io/istio/operator/pkg/util"
```
Comment (tariq1890) [inline operator/pkg/tpath/struct_test.go:22]: As per Go convention, this is the order of imports

i) stdlib
ii) third-party imports
iii) first-party imports
```suggestion
	"testing"

	"github.com/ghodss/yaml"

	"istio.io/istio/operator/pkg/util"
```

**Review by morvencao — APPROVED** [review]

File: operator/pkg/tpath/tree_test.go [hunk 2]
```diff
@@ -15,11 +15,11 @@
 package tpath
 
 import (
-	"testing"
-
 	"github.com/ghodss/yaml"
 
 	"istio.io/istio/operator/pkg/util"
+
+	"testing"
```
Comment (tariq1890) [inline operator/pkg/tpath/tree_test.go:22]: Thanks @adiprerepa . I believe you missed this spot.

Comment (adiprerepa) [reply operator/pkg/tpath/tree_test.go:22]: oops - thanks.

Comment (adiprerepa) [reply operator/pkg/tpath/tree_test.go]: should be good now @tariq1890

**Review by tariq1890 — APPROVED** [review]

**Pull request merged**